File: flutter_baidu_yingyan_trace-1.0.0/android/src/main/java/com/baidu/flutter/trace/model/entity/DeleteEntityResult.java
package com.baidu.flutter.trace.model.entity;

import com.baidu.trace.model.BaseResponse;

/**
 * 删除Entity响应结果
 *
 * @author baidu
 */
public final class DeleteEntityResult extends BaseResponse {

    /**
     * @param tag     响应标识（与请求标识一致）
     * @param status  状态码
     * @param message 状态消息
     */
    public DeleteEntityResult(int tag, int status, String message) {
        super(tag, status, message);
    }

    @Override
    public String toString() {
        return "DeleteEntityResult [tag=" + tag + ", status=" + status + ", message=" + message
                + "]";
    }

}

File: flutter_baidu_yingyan_trace-1.0.0/lib/src/model/entity/delete_entity_result.dart
import 'package:flutter_baidu_yingyan_trace/src/model/base_result.dart';

/// 删除Entity响应结果
class DeleteEntityResult extends BaseResult {
  DeleteEntityResult({required int tag, required int status, required String message})
      : super(tag: tag, status: status, message: message);

  /// map => DeleteEntityResult
  DeleteEntityResult.fromMap(Map map)
      : assert(map != null,
            'Construct a DeleteEntityResult，The parameter map cannot be null !'),
        super.fromMap(map);

  @override
  Map toMap() {
    return super.toMap();
  }

  @override
  String toString() {
    return "DeleteEntityResult{tag = $tag; status = $status; message = $message}";
  }
}
